test/integration/local_transactions_test.rb

```diff
@@ -403,91 +405,180 @@ class LocalTransactionsTest < ActionDispatch::IntegrationTest
+            "type" => "LGD",
+            "govuk_slug" => "cardiff",
+            "country_name" => "Wales",
+          },
+        ],
+      )
 
-        visit "/pay-bear-tax"
-        fill_in "postcode", with: "WA8 8DX"
-        click_on "Find"
-      end
+      stub_local_links_manager_has_no_link_and_no_homepage_url(
+        authority_slug: "cardiff",
+        lgsl: 461,
+        lgil: 8,
+        country_name: "Wales",
+      )
 
-      should "render the service unavailable in country page" do
-        assert page.has_content? "Pay your bear tax"
-      end
+      visit "/pay-bear-tax"
+      fill_in "postcode", with: "WA8 8DX"
+      click_on "Find"
+    end
 
-      should "show a button that links to an appropriate alternate service provider" do
-        assert_has_button_as_link(
-          "Find other services",
-          href: "http://cardiff.example.com", # local authority link from stubbed local links
-          rel: "external",
-          start: true,
-        )
-      end
+    should "render unavailable_service view" do
+      assert page.has_content? "This service is not available in Wales."
+    end
+
+    should "render matching the postcode to local authority" do
+      content = "We've matched the postcode to <span class=\"local-authority\">Cardiff</span>."
+      assert page.body.include?(content)
+    end
+
+    should "not link to the authority" do
+      assert page.has_no_link?("Go to their website")
+    end
+
+    should "show advisory message that we have no url" do
+      assert page.has_content?("We don't have a link for their website.")
+      assert page.has_link?("local council search", href: "/find-local-council")
+    end
+  end
+
+  context "when a service is available for a devolved administration" do
+    setup do
+      stub_mapit_has_a_postcode_and_areas(
+        "EH8 8DX",
+        [0, 0],
+        [
+          {
+            "name" => "Holyroodhouse",
+            "type" => "LGD",
+            "govuk_slug" => "edinburgh",
+            "country_name" => "Scotland",
+          },
+        ],
+      )
+
+      stub_local_links_manager_has_a_link(
+        authority_slug: "edinburgh",
+        lgsl: 461,
+        lgil: 8,
+        url: "http://www.edinburgh.gov.uk/bear-the-cost-of-grizzly-ownership",
+        country_name: "Scotland",
+        status: "ok",
+      )
+
+      visit "/pay-bear-tax"
+      fill_in "postcode", with: "EH8 8DX"
+      click_on "Find"
+    end
+
+    should "render devolved_administration_service view" do
+      assert page.has_content? "Get information on this service, or a similar service, on the Scotland Government website"
+    end
+
+    should "show a button that links to an alternate service provider" do
+      assert_has_button_as_link(
+        "Find information for Scotland",
+        href: "https://scot.gov/service",
+        rel: "external",
+      )
+    end
+  end
+
+  context "when availability data is not provided for a devolved administration" do
```

Comment:
I don't think we need this test as it doesn't exercise any different code paths. I think we should just make sure one of the earlier integration tests of the feature has a non England country name so we can implicitly cover it.

Reply:
I removed it. I guess having the other tests for devolved administrations / the Westminster postcode for England for the tests at the top of the file cover this.